eskipfile/file.go

```diff
@@ -45,9 +45,16 @@ func Open(path string) (*Client, error) {
 	return &Client{routes}, nil
 }
 
+func (c Client) LoadAndParseAll() (routeInfos []*eskip.RouteInfo, err error) {
+	for _, route := range c.routes {
+		routeInfos = append(routeInfos, &eskip.RouteInfo{*route, nil})
```

Comment:
similarly to the other places regarding the WriteClient and ReadClient interfaces, this feels a bit of a type-fu.